evaluators/embedding/L1NormEvaluator/__init__.py

```diff
@@ -7,15 +10,16 @@ class L1NormEvaluator(BaseEmbeddingEvaluator):
     """
     :class:`L1NormEvaluator` evaluates the distance between actual
     and desired embeddings computing the L1 Norm between them
+
+    :param actual: the embedding of the document
+        (resulting from an Encoder)
+    :param desired: the expected embedding of the document
+    :param args: additional positional arguments.
+    :param kwargs: additional positional arguments.
+    :return: the evaluation metric value for the request document
```

Comment:
These are the docstrings of method not init() I think. So should keep them under def evaluate()?

Reply:
Yes, makes sense, I'm moving it back under the `evaluate` method